zookeeper-server/src/main/java/org/apache/zookeeper/util/SecurityUtils.java

```diff
@@ -36,10 +42,14 @@
 import org.ietf.jgss.GSSName;
 import org.ietf.jgss.Oid;
 import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public final class SecurityUtils {
 
+    private static final Logger LOG = LoggerFactory.getLogger(SecurityUtils.class);
```

Comment:
How about follow the pattern of `createSaslClient`/`createSaslServer` to accept a log parameter ? This will allow callers to log in their contexts.